MediaRecorder.html

```diff
@@ -442,6 +443,38 @@
               <em>Return type:</em> <code>boolean</code>
             </div>
           </dd>
+
+          <dt><dfn><code>getSupportedTypes</code></dfn>, static</dt>
+          <dd>
+            When <code>getSupportedTypes()</code> method is invoked, the UA MUST
+            queue a task, using the DOM manipulation task source, that runs the
+            following steps:
+            <ol>
+            <li>Let <var>p</var> be a promise.</li>
+            <li>Run the following steps in parallel:
+             <ol>
+             <li>Let <var>resultList</var> be an empty list.
+             <li>For each <a href="https://tools.ietf.org/html/rfc2046">MIME
```

Comment:
Hmm, so from that (or any other spec) would you say that "video/webm;codecs=vp8" or "video/webm; codecs=vp8" is the expected string from this API? If it is actually defined, it would still be good to call out in a non-normative note what is expected and why, so that there can't be any divergence on silly things like whitespace, capitalization or quotes.